specification/communication/data-plane/PhoneNumbers/preview/2021-10-25-preview/phonenumbers.json

```diff
@@ -0,0 +1,1639 @@
+            "schema": {
+              "$ref": "#/definitions/PurchasedPhoneNumbers"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "phoneNumbers"
+        },
+        "x-ms-examples": {
+          "List purchased phone numbers": {
+            "$ref": "./examples/GetPhoneNumbers.json"
+          }
+        }
+      }
+    },
+    "/shortCodes": {
+      "get": {
+        "tags": [
+          "ShortCodes"
+        ],
+        "summary": "Gets the list of short codes for the current resource.",
+        "operationId": "ShortCodes_GetShortCodes",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "An optional parameter for how many entries to skip, for pagination purposes.",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "take",
+            "description": "An optional parameter for how many entries to return, for pagination purposes.",
+            "type": "integer",
+            "format": "int32",
+            "default": 100
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ShortCodeEntities"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "shortCodes"
+        },
+        "x-ms-examples": {
+          "List short codes": {
+            "$ref": "./examples/GetShortCodesAsync.json"
+          }
+        }
+      }
+    },
+    "/shortCodes/countries/US/programBriefs/{programBriefId}": {
+      "patch": {
+        "tags": [
+          "ShortCodes - ProgramBrief"
+        ],
+        "summary": "Creates or updates a US Program Brief.",
+        "operationId": "ShortCodes_UpsertUSProgramBrief",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "programBriefId",
+            "description": "Program Brief Id. Must be a valid GUID",
+            "required": true,
+            "type": "string"
```

Comment:
Nit - consider using `"format": "uuid"` if these must be GUIDs.

Reply:
Good catch. I did this for the ProgramBriefEntity model type. Will change it on the path params